src/controllers/controller.line.js

```diff
@@ -106,15 +103,15 @@ module.exports = DatasetController.extend({
 	},
 
 	updateElement: function(point, index, reset) {
-		var me = this;
-		var meta = me.getMeta();
-		var xScale = me._xScale;
```

Comment:
do we still need to set `_xScale` since it's not used here?

Reply:
Sorry, I'm not sure what you mean?

Reply:
Nevermind. You already addressed it